code/datums/components/chasm.dm

```diff
@@ -85,10 +85,15 @@
 			var/mob/living/carbon/human/H = AM
 			if(istype(H.belt, /obj/item/wormhole_jaunter))
 				var/obj/item/wormhole_jaunter/J = H.belt
-				//To freak out any bystanders
-				H.visible_message(span_boldwarning("[H] falls into [parent]!"))
-				J.chasm_react(H)
-				return FALSE
+				var/list/D = J.get_destinations()
+
+				if(D.len)
+					to_chat(H, span_notice("Your [J.name] activates, saving you from the chasm!"))
+					H.visible_message(span_boldwarning("[H] falls into [parent]!")) //To freak out any bystanders
+					J.chasm_react(H)
+					return FALSE
+				else if(!D.len)
+					to_chat(H, span_userdanger("The [J.name] found no beacons in the world to anchor a wormhole to, preventing it from saving you from the chasm.  RIP."))
```

Comment:
Why not have the jaunter itself say something like "No beacons detected" rather then a status message?

Reply:
Also why do this here and not on the jaunter?

Reply:
So the jaunter is in the player's belt slot when this is triggered.  I used the original message that previously existed.  It was just moved from jaunter to the chasm.  The main reason we are doing this in the chasm is because if the beacons don't exist then the jaunter is not triggered and is ignored.  This returns FALSE for the chasm function which determines if the mob will fall.  

When this code logic was apart of the jaunter code initially there was a nasty bug where no destinations would result in the player not ever falling because it wasn't returning FALSE in the chasm proc.  That's why it was moved.  Otherwise a player could freely walk over chasms without the jaunter being triggered or falling.

Reply:
I'd prefer if the jaunter proc chain coming off `chasm_react()` just returned TRUE or FALSE if it did or didn't do anything, so this issue doesn't reproduce when some guy adds like, another way to disable the things. 

That and we don't have to duplicate the destinations check inside the jaunter and inside the chasm that way

Reply:
I applied the necessary changes with https://github.com/tgstation/tgstation/pull/61055/commits/391daa0bc60c491f6e62e62498b2bdd8005df87b